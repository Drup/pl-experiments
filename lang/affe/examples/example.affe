let id = fun x -> x
let id2 = id id
let x =
  let x = id2 3 in
  id2
let a = 2
let plusa = fun b -> a + b
let c = plusa 2
