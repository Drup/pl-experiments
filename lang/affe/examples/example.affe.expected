id : âˆ€^k, 'a. ('a : ^k) => 'a -> 'a

id2 : ('_a -> '_a)

x : int -> int

a : int

plusa : int -> int

c : int

